build.gradle

```diff
@@ -282,7 +282,7 @@ dependencies {
   integrationTestImplementation sourceSets.main.runtimeClasspath
   integrationTestImplementation sourceSets.test.runtimeClasspath
 
-  testImplementation group: 'uk.gov.hmcts', name: 'befta-fw', version: '5.9.0'
+  testImplementation group: 'uk.gov.hmcts', name: 'befta-fw', version: '5.19.0-RDM-9893-3'
```

Comment:
Are these changes related or should they be in a separate review ?

Reply:
There were updates to the spreadsheets required as part of the changes, so created a pre-release version - will be set appropriately with a complete release as part of the standard QA process 👍